### PR TITLE
Update data

### PR DESCRIPTION
**Pull request by hofi1**

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -1165,18 +1165,12 @@ https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,extensi
 https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,extensions-core/kafka-indexing-service,org.apache.druid.indexing.kafka.supervisor.KafkaSupervisorTest.testReportWhenMultipleActiveTasks,ID,,,
 https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,extensions-core/kinesis-indexing-service,org.apache.druid.indexing.kinesis.supervisor.KinesisSupervisorTest.testIsTaskCurrent,ID,,,
 https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,extensions-core/kinesis-indexing-service,org.apache.druid.indexing.kinesis.supervisor.KinesisSupervisorTest.testShardSplit,ID,,,
-https://github.com/apache/druid,4b0d1b3488add75097532f25c8bbbcb35daa4987,processing,org.apache.druid.java.util.emitter.core.EmitterTest.testBasicAuthenticationAndNewlineSeparating,ID,,,
-https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,core,org.apache.druid.java.util.emitter.core.EmitterTest.testBasicAuthenticationAndNewlineSeparating,ID,MovedOrRenamed,,https://github.com/apache/druid/commit/08b5951cc53c4fe474a129500c62a6adad78337f
```
Comment (darko-marinov) [inline pr-data.csv:1169]: Don't remove `MovedOrRenamed`.

File: pr-data.csv [hunk 2]
```diff
@@ -1165,18 +1165,12 @@ https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,extensi
...
+https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,processing,org.apache.druid.java.util.emitter.core.EmitterTest.testBatchSplitting,ID,Opened,https://github.com/apache/druid/pull/15261,
+https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,processing,org.apache.druid.java.util.emitter.core.EmitterTest.testGzipContentEncoding,ID,Opened,https://github.com/apache/druid/pull/15261,
+https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,processing,org.apache.druid.java.util.emitter.core.EmitterTest.testSanity,ID,Opened,https://github.com/apache/druid/pull/15261,
+https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,processing,org.apache.druid.java.util.emitter.core.EmitterTest.testSanityWithGeneralizedCreation,ID,Opened,https://github.com/apache/druid/pull/15261,
+https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,processing,org.apache.druid.java.util.emitter.core.ParametrizedUriEmitterTest.testEmitterWithMultipleFeeds,ID,MovedOrRenamed,,https://github.com/apache/druid/commit/08b5951cc53c4fe474a129500c62a6adad78337f
```
Comment (darko-marinov) [inline pr-data.csv:1173]: Should this line be `Opened` (with PR link) not `MovedOrRenamed`?

**Review by darko-marinov — CHANGES_REQUESTED** [review]
Check `MovedOrRenamed` carefully.

**Comment by hofi1** [comment]
Looks like something went wrong while rebasing – should be fixed now

**Pull request merged**